File: src/main/java/com/train/Tester.java
package com.train;

import org.jetbrains.annotations.NotNull;

import java.util.Scanner;

public class Tester {
    public static void main(String[] args) {
        Ticket ticket = getTicket();
        ticket.count();
    }

    @NotNull
    private static Ticket getTicket() {
        System.out.println("Please enter number of tickets:");
        Scanner scanner=new Scanner(System.in);
        double tickets=scanner.nextDouble();
        System.out.println("How many round-trip tickets:");
        double roundtrip =scanner.nextDouble();
        return new Ticket(tickets,roundtrip);
    }
}

File: src/main/kotlin/com/train/Tester.kt
package com.train

import java.util.*

fun main(args: Array<String>) {
    val scanner=Scanner(System.`in`)
    print("please enter number of ticketes")
    var ticket=scanner.nextDouble()
    print("please enter number of ticketes")
    var roundtrip=scanner.nextDouble()
    val train=Trainticket(ticket,roundtrip);
    train.count()
}
class Trainticket(var ticket:Double, var roundtrip:Double){

fun count(){
    var total:Double
    total=(ticket-roundtrip)*1000+(roundtrip*2000*0.9)
    println("Total ticketes"+ticket)
    println("Round-trip"+roundtrip)
    println("Total price  "+total)


}
}
